ui/pages/Public/components/FeatureUpdates.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react'
+import { Feed, Header } from 'semantic-ui-react'
+import TextFieldView from 'shared/components/panel/view-fields/TextFieldView'
+import { HttpRequestHelper } from 'shared/utils/httpRequestHelper'
+
+const FeatureUpdates = () => {
```

Comment:
So this might be out of scope for this PR, but the functionality in the component seems very similar to that of the `StateDataLoader` component,  in that that is a component that fetches some initial data, shows a loading spinner while fetching, shows an error message if an error occurs, and otherwise renders a child component with props that were fetched. For consistencies sake, I think it might be good if this component and that component ended up sharing functionality. If you want to take this live as-is and then do a follow up to make shared behavior that would be fine with me

Reply:
made a few minor changes to use StateDataLoader